samples/explore_datasource.py

```diff
@@ -67,6 +67,17 @@ def main():
             print(["{0}({1})".format(connection.id, connection.datasource_name)
                    for connection in sample_datasource.connections])
 
+            # Add some tags to the datasource
+            server.version = 2.6 # Datasource tagging requires server version 2.6
```

Comment:
It's not safe to just set this to 2.6, it'll cause all non 2.6 Servers to fail even on sign in.

That actually reminds me that maybe all samples should call `server.use_highest_version()`